finance/src/validations/payment.validations.js

```diff
@@ -0,0 +1,46 @@
+const JOI = require('joi');
+const HTTPStatus = require('../helpers/HTTP.status');
+const customError = require('../helpers/error.custom');
+
+const payloadValidation = (payload) => {
+  const { error } = JOI.object({
+    value: JOI.number().min(1).required(),
+    buyerName: JOI.string().required(),
+    cardNumber: JOI.number().required(),
+    expirationDate: JOI.date().required(),
+    cvv: JOI.number().required(),
+  }).validate(payload);
+
+  if (error) throw customError(error.message, HTTPStatus.UN_ENTITY);
+  return null;

```

Comment:
Não é necessário retornar null em uma função JS, em especial se o retorno não for utilizado.